### ISSUE TITLE
Not working help

### ISSUE THREAD

**Issue by Stonks476** [DrNoLife/Destiny-2-Solo-Enabler#32]
so ive download files and it just says 

![Capture1](https://user-images.githubusercontent.com/114644243/192954742-a4ab25af-5edd-4842-920d-ee2d0fae17bd.PNG)
![Capture](https://user-images.githubusercontent.com/114644243/192954760-2f716df1-7123-4794-ab87-7d5d51c2dd8a.PNG)

and is my folder meant to look like this if you could help that would be great

**Comment by BLTplayz** [comment]
Hello @Stonks476 , I believe you downloaded the source code rather than the executable. Simply go to [this](https://github.com/DrNoLife/Destiny-2-Solo-Enabler/releases/tag/2.4) link and click Destiny.2.Solo-Enabler.exe. This should download the executable and you should be able to start queuing solo!

**Comment by DrNoLife** [comment]
As @BLTplayz said, you downloaded the incorrect files.

TL;DR: Follow the link BLT provided, and download the latest .exe. There's also a video guide on the subject. https://www.youtube.com/watch?v=yunmINhf6c4

Now, to explain what you did download. You got the source code of the program.
The program is open-source, meaning it's freely available for everyone, completely for free, with the possibility of going through the entire code-base to see what the program is doing. 

One would download the source code if they wanted to view it in their own IDE (basically, fancy text editor for programming stuff), or if they wanted to try and make changes to it. They could also download the source-code and compile the program themself. Some people do this, because they want to be sure that the exe released matched the code published.

Hope this cleared it up.

**Comment by Stonks476** [comment]
cheers thanks for the help!
